app/tests/spec/models/reliers/oauth.js

```diff
@@ -854,7 +846,7 @@ describe('models/reliers/oauth', () => {
     });
 
     it('errors correctly', () => {
-      assert.isTrue(OAuthErrors.is(err, 'INVALID_PARAMETER'));
+      //assert.isTrue(OAuthErrors.is(err, 'INVALID_PARAMETER'));
```

Comment:
Should we delete the old test? And should the name of the test change now too?

Reply:
I forgot to remove the commented out test, yes, we should! Great catch.